docs/testing.md

```diff
@@ -1,35 +1,35 @@
-# Testing
+# Compile and run tests
 
 ## Docker image access
 
 In order to run tests, custom docker images are needed. They can be build locally, but they are also available on the
 GitHub Dockerhub repository. Inorder to use the Docker images from this repository one need to be logged. Follow the
 [official guidelines](https://docs.github.com/en/packages/guides/configuring-docker-for-use-with-github-packages#authenticating-to-github-packages) from GitHub to login to your account.
 
+## Repository location
+
+Before trying to run tests locally one have to define the environment variable that will point to
+root folder of this repository as an absolute path on file system. It is needed as running the tests from IDE
+have the other relative home path than when running on CI pipeline. This can be done from terminal by running
+```shell
+export REPOSITORY_ABSOLUTE_ROOT=$(pwd)
+```
+from repository root folder or by defining the environment variable in the IDE with proper absolute path to repository root.
+
 ## Command line
 
-Tests can be easily run using the _sbtwrapper_ and the commands the same as in the [unit_tests.yaml]({{ book.sourcesRootUrl }}.github/workflows/unit_tests.yaml)
+Tests can be easily run using the `sbtwrapper` and the commands the same as in the `unit_tests.yaml`
```

Comment:
You can do both preformatted text and links in Markdown. I think we should link source files or at least use full paths, since readers won't know which file it is.

Reply:
ok, done